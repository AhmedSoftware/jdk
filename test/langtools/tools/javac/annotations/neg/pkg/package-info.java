/*
 * @test /nodynamiccopyright/
 * @bug 8028389
 * @summary javac should output a proper error message when given something
 * like new Object(){} as annotation argument.
 *
 * @compile AnonSubclassOnPkg.java
 * @compile -J-XX:+UnlockExperimentalVMOptions -J-XX:hashCode=2 AnonSubclassOnPkg.java
 * @compile/fail/ref=package-info.out -XDrawDiagnostics package-info.java
 */

@AnonSubclassOnPkg(new Object(){})
package pkg;
