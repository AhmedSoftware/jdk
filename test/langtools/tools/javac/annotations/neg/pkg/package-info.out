package-info.java:12:20: compiler.err.expression.not.allowable.as.annotation.value
1 error
